### PR TITLE
#9 appbar

### PR DESCRIPTION
**Pull request by tymwitko**

### CODE REVIEW

**Review by Kszesiek — CHANGES_REQUESTED** [review]

File: mobile/assets/temp.png
[the remark this answers is not in the document]
Comment (tymwitko) [reply mobile/assets/temp.png:1]: Right, deleted

File: mobile/lib/components/ra_appbar.dart [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  final IconButton? iconButton;
+  final String text;
+  final String? iconPath;
+  final EdgeInsets? titlePadding;
+  final int imageHeight;
+
+  @override
+  Widget build(BuildContext context) {
+    return AppBar(
+      toolbarHeight: height.toDouble(),
```
[the remark this answers is not in the document]
Comment (tymwitko) [reply mobile/lib/components/ra_appbar.dart]: Done

File: mobile/lib/components/ra_appbar.dart [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      bottom: PreferredSize(
+        preferredSize: const Size.fromHeight(0),
+        child: Container(
+          color: accentColor,
+          height: bottomSize.toDouble(),
+        ),
+      ),
+      actions: <Widget>[
+        iconButton!
```
[the remark this answers is not in the document]
Comment (tymwitko) [reply mobile/lib/components/ra_appbar.dart]: Done

File: mobile/lib/components/ra_appbar.dart [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+      ],
+      backgroundColor: mainColor,
+      title: Padding(
+        padding: titlePadding!,
+        child: Row(
+          children: [
+            SvgPicture.asset(
+              height: imageHeight.toDouble(),
+              iconPath!,
```
[the remark this answers is not in the document]
Comment (tymwitko) [reply mobile/lib/components/ra_appbar.dart:57]: Done with null safety

**Review by Kszesiek — APPROVED** [review]
Looks right

**Pull request merged**